internal/console/console.go

```diff
@@ -80,9 +81,10 @@ type Console struct {
 }
 
 // New returns a new Console object that uses a RPC client to get information from a torrent.Session.
-func New(clt *rainrpc.Client) *Console {
+func New(clt *rainrpc.Client, columns []string) *Console {
 	return &Console{
 		client:          clt,
+		columns:         columns,
```

Comment:
Set the defaults here.

Reply:
Is it necessary to set it here as well? It will already get a default value (`"# ID Name"`) from `main.go` and it might get confusing to have it in two places if it ever changes.

Reply:
Since this is an internal package, I think we don't need to preserve backwards compatibility here. I am going to merge this PR and revert your last commit.